app/controllers/tag_layout_templates_controller.rb

```diff
@@ -23,7 +23,8 @@ def show
   ##
   # Allows for the passing in of tag group id using a link from the tag group show page.
   def new
-    @tag_layout_template = TagLayoutTemplate.new(tag_group_id: params[:tag_group_id])
+    @tag_layout_template =
+      TagLayoutTemplate.new(tag_group_id: params[:tag_group_id], tag2_group_id: params[:tag2_group_id])
```

Comment:
What happened here, was this a bug?

Reply:
This is just to facilitate auto populating both the tag_group and tag2_group in a tag layout template when navigating from a TagSet as previously it was only navigated to from a tag group so a tag2_group param wasn't useful.